mode/src/processing/mode/android/SDKDownloader.java

```diff
@@ -192,22 +192,18 @@ protected Object doInBackground() throws Exception {
         
          // Normalize built-tools and platform folders to android-<API LEVEL>
         String actualName = platformsFolder.listFiles()[0].getName();
-        renameFolder(platformsFolder, "android-" + AndroidBuild.TARGET_SDK, actualName);
+        renameFolder(platformsFolder, "android-" + TARGET_SDK, actualName);
         actualName = buildToolsFolder.listFiles()[0].getName();
         renameFolder(buildToolsFolder, downloadUrls.buildToolsVersion, actualName);
         
         // Done, let's set the environment and load the new SDK!
         Platform.setenv("ANDROID_SDK", sdkFolder.getAbsolutePath());
         Preferences.set("android.sdk.path", sdkFolder.getAbsolutePath());        
         sdk = AndroidSDK.load(false, null);        
-      } catch (ParserConfigurationException e) {
+      } catch (IOException e) {
```

Comment:
I noticed you changed the multiple catches for a single catch. Can you comment in your decision? @DeerajTheepshi

Reply:
I haven't replaced them completely. All the removed exceptions, ie..ParserConfigurationException,SAXException, are thrown only by the XML parsing process that happens on getMainDownloadUrls() or getExtrasDownloadUrls(). Since they have been moved from the swing worker, these are not handled here. However you can find them at line 612 of the same class.